### ISSUE TITLE
Add Figshare as data source

### ISSUE THREAD

**Issue by flaneuse** [biothings/metadataplus#10]
## What is the source's name
Figshare

## What is the web address of the source
[General Figshare instance](https://figshare.com/search?searchMode=1&types=3)
[https://nih.figshare.com/](NIH instance)

Note 1: It appears that all datasets within the NIH instance are also available in the general Figshare repo (3/3 that I checked), but that *is not* confirmed (@andrewsu do you have insider info?).  It'd be nice to keep track of whether the dataset is in the NIH portion of Figshare or just the general repo.

Note 2: There are lots of lots of types of data that Figshare publishes (figures, datasets, software, etc.).  Only the datasets should be scraped.

## Example web address of one dataset
https://nih.figshare.com/articles/Genome-wide_summary_statistics_used_for_the_analysis_described_in_Rare_protein-altering_variants_in_ANGPTL7_lower_intraocular_pressure_and_protect_against_glaucoma_/11368022

## Does this source require a user account login? Y/N

- [ ] Yes
- [x] No

## In short describe the importance of this source's addition
General purpose repo for uploading datasets, amongst other things.  NIH has its own dedicated instance.
